software/SolarAISVN/SolarAISVN.ino

```diff
@@ -1,49 +1,65 @@
-// Solarmeter first attempt
-// Inspired by https://randomnerdtutorials.com/esp32-esp8266-publish-sensor-readings-to-google-sheets/
+// Solar- and windmeter at AISVN v0.5 2020/06/08
  
```

Comment:
Good to have the names updated. And version included, as well as revision number.